python/pyspark/sql/observation.py

```diff
@@ -112,12 +115,13 @@ def get(self):
 
         Returns
         -------
-        :class:`Row`
+        :class:`Dict`
             the observed metrics
         """
         assert self._jo is not None, 'call DataFrame.observe'
-        jrow = self._jo.get()
-        return self._to_row(jrow)
+        jmap = self._jo.getAsJavaMap()
+        # return a pure Python dict, not a py4j JavaMap
+        return {k: v for k, v in jmap.items()}
```

Comment:
can you actually directly return `self._jo.getAsJavaMap()`. That becomes a plain dictionary IIRC.

Reply:
no, that is a JavaMap, a py4j implementation of dict that syncs changes of the Python dict back to the Java map inside the jvm

Reply:
👌